Samples/native/example/example/ContentView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ContentView.swift
+//  example
+//
+//  Created by Brian Smith on 11/30/22.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
```

Comment:
Is this file for the host app for the live activity?

Reply:
This file is for the "dummy host app" it is required for the Xamarin solution to properly pull in the widget extension.  It is not actually used.